discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -52,6 +52,22 @@ def _create_tf_folder(self, name: str, platform: str):
 
         return os.path.join(tf_folder, consts.Platforms.BARE_METAL)
 
+    def _get_disk_encryption_appliance(self):
```

Comment:
What about `VSphereController`? We don't want to support it in [vsphere](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-3D39CBA6-E5B2-43E2-A596-B9A69B094558.html)?

Reply:
The emulator only works on libvirt